pyfarm/master/user_interface/software.py

```diff
@@ -0,0 +1,191 @@
+# No shebang line, this module is meant to be imported
+#
+# Copyright 2015 Ambient Entertainment GmbH & Co. KG
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+try:
+    from httplib import NOT_FOUND, SEE_OTHER, BAD_REQUEST
+except ImportError:  # pragma: no cover
+    from http.client import NOT_FOUND, SEE_OTHER, BAD_REQUEST
+
+from flask import render_template, request, redirect, url_for, flash
+
+from sqlalchemy import or_
+from sqlalchemy.sql.expression import func
+
+from pyfarm.models.software import (
+    Software, SoftwareVersion, JobSoftwareRequirement,
+    JobTypeSoftwareRequirement)
+from pyfarm.master.application import db
```

Comment:
It's a nit so I'm not going to push on this unless there's a good reason to (and there's not).  I've tried to structure most other imports alphabetically but consistency is more important so I'd keep this the way it is since that's how the other user_interface modules are doing it.